File: dotnet/src/Model/World/Water.cs
using System;

namespace PlanetCS1 {
    class Water : RealTile {
        public Water(Location location) {
            this.loc = location;
            this.occ = false;
            this.use = false;
            this.drive = false;
            this.traverse = false;
            this.owner = 0;
            this.icon = "///";
            this.minCost = 0;
            this.gasCost = 0;
            this.nrgCost = 0;
            this.health = 0;
            this.minSal = 0;
            this.gasSal = 0;
            this.nrgSal = 0;
        }
    }
}

File: kotlinjs/old/Water.kt


class Water(location: Location) : RealTile() {
    override var loc = location
    override var occ = false
    override var use = false
    override var drive = false
    override var traverse = false
    override var owner = 0
    override var icon = "///"
    override var minCost = 0
    override var gasCost = 0
    override var nrgCost = 0
    override var health = 0
    override var minSal = 0
    override var gasSal = 0
    override var nrgSal = 0
}